src/it/java/servlets/admin/config/EnableFeedbackIT.java

```diff
@@ -1,258 +1,258 @@
-			{
-				String message = new String("No CSRF token returned from Login Servlet");
-				log.fatal(message);
-				fail(message);
-			}
-			else
-			{
-				//Add Cookies from Response to outgoing request
-				request.setCookies(response.getCookies());
-				String responseBody = doThePost(csrfToken);
-				if(responseBody.contains("Feedback Enabled"))
-				{
-					log.debug("Feedback Enabled");
-					if(FeedbackStatus.isDisabled())
-					{
-						String message = "Feedback was not actually Enabled for players";
-						log.fatal(message);
-						fail(message);
-					}
-				}
-				else
-				{
-					String message = "Admin was unable to enable Feedback";
-					log.fatal(message);
-					fail(message);
-				}
-			}
-		}
-		catch (Exception e)
-		{
-			log.fatal("Could not Complete: " + e.toString());
-			fail("Exception Caught: " + e.toString());
-		}
-	}
-
-	@Test
-	public void testCsrf()
-	{
-		String userName = "configAdminTester";
-		String password = userName;
-		//Verify / Create user in DB
-		try
-		{
-			TestProperties.verifyTestAdmin(log, applicationRoot, userName, password);
-			//Sign in as Admin User
-			log.debug("Signing in as User Through LoginServlet");
-			TestProperties.loginDoPost(log, request, response, userName, userName, null, lang);
-			log.debug("Login Servlet Complete, Getting CSRF Token");
-			if(response.getCookie("token") == null)
-				fail("No CSRF Token Was Returned from Login Servlet");
-			String csrfToken = response.getCookie("token").getValue();
-			if(csrfToken.isEmpty())
-			{
-				String message = new String("No CSRF token returned from Login Servlet");
-				log.fatal(message);
-				fail(message);
-			}
-			else
-			{
-				//Add Cookies from Response to outgoing request
-				request.setCookies(response.getCookies());
-				String responseBody = doThePost("wrongToken");
-				if(responseBody.contains("Enable Feedback Failure"))
-				{
-					log.debug("CSRF Error Occurred");
-					if(FeedbackStatus.isEnabled())
-					{
-						String message = "Feedback was enabled after what should have been a failed request";
-						log.fatal(message);
-						fail(message);
-					}
-				}
-				else
-				{
-					String message = "No CSRF Error Detected";
-					log.fatal(message);
-					fail(message);
-				}
-			}
-		}
-		catch (Exception e)
-		{
-			log.fatal("Could not Complete: " + e.toString());
-			fail("Exception Caught: " + e.toString());
-		}
-	}
-}
+package servlets.admin.config;
+
+import static org.junit.Assert.*;
+
+import org.apache.log4j.Logger;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.mock.web.MockServletConfig;
+
+import testUtils.TestProperties;
+import utils.FeedbackStatus;
+import utils.InstallationException;
+
+public class EnableFeedbackIT
+{
+	private static org.apache.log4j.Logger log = Logger.getLogger(EnableCheats.class);
```

Comment:
This should also be renamed to EnableFeedbackIT (EnableCheats.class is a mistake from original)